documentation/adr/0002-use-github-for-documentation.md

```diff
@@ -16,14 +18,23 @@ Documentation will directly reference Horace and Herbert projects.
 
 The GitHub Wiki repository does not support pull requests, or the display of any content not on the the `master` branch.
 
+
 ## Decision
 
-We will store documentation in the `Horace/documentation` folder and link to this from the the [GitHub Wiki](https://github.com/pace-neutrons/pace-developers/wiki) which is attached to the PACE project on GitHub. 
+~We will store documentation in the `Horace/documentation` folder and link to this from the [GitHub Wiki](https://github.com/pace-neutrons/pace-developers/wiki) which is attached to the PACE project on GitHub.~ 
+
+We will store documentation in the `Horace/documentation` folder which will be built regularly and deployed to the GitHub pages under the following scheme: 
+
+The documentation will be supported and built by the Jenkins pipeline and automatically deployed on:
+- nightly builds (unstable)
+- release pipeline (stable & horace-version)
```

Comment:
The release pipelines build release packages, but we don't necessarily release those packages (we do some testing first). Should the Deploy pipeline or some GitHub action deploy releases?